src/SimpleWebRequest.ts

```diff
@@ -12,16 +12,20 @@ import SyncTasks = require('synctasks');
 
 import { ExponentialTime } from './ExponentialTime';
 
+export interface Headers {
+    [header: string]: string|undefined;
```

Comment:
Change back to just `string` instead of `string|undefined`. Sorry to go back on what we said previously. This causes a handful of compiler errors, and although they are legitimate, I don't think this PR is the right place to introduce them. We will change this later.

If you also add the default to the type parameter below then this PR should not have breaking changes, which would be nice.